Library/Homebrew/style.rb

```diff
@@ -231,6 +214,20 @@ def run_shellcheck(files, output_type)
       end
     end
 
+    def run_shfmt(files, inplace: false)
+      files = shell_scripts if files.empty?
+      # Do not format completions and Dockerfile
+      files.delete(HOMEBREW_REPOSITORY/"completions/bash/brew")
+      files.delete(HOMEBREW_REPOSITORY/"Dockerfile")
+
+      args = ["-i", "2", "-ci", "-ln", "bash", "--", *files]
+
+      args.unshift("-w") if inplace
```

Comment:
```suggestion
      args << "-w" if fix
```

Reply:
I find that we should add `"-w"` flag before `"--"` here.